### DISCUSSION TITLE
pocketbase field type for autoincrement

### DISCUSSION THREAD

**Discussion by bobwatcherx** [pocketbase/pocketbase#4891] [Q&A]
is there field for auto increment . 
![Screenshot 2024-05-10 152525](https://github.com/pocketbase/pocketbase/assets/52072811/ea29742b-3b7f-4567-be66-55565b84a227)

let's say I want to create an invoice code for example

inv-000
inv-001
inv-002
etc

what type of field to do that

**Comment by ganigeorgiev** [comment]
All available field types are listed in the shown dropdown. There is no autoincrement field type at the moment and there are no plans to add such out of the box as the counter requirement will differ from one use case to another (eg. you may want the counter to be per "group") and it is better left to users to manage it on their own if there is really need for it.
